error_pages/404.php

```diff
@@ -2,118 +2,131 @@
-		<header id="header" role="banner" class="wdn-content-slide wdn-band">
-			<div id="wdn_header_top">
-				<span id="wdn_institution_title"><a href="https://www.unl.edu/">University of Nebraska&ndash;Lincoln</a></span>
-				<div id="wdn_resources">
-					<?php wdnInclude("/wdn/templates_4.1/includes/wdnResources.html"); ?>
-					<?php wdnInclude("/wdn/templates_4.1/includes/idm.html"); ?>
-					<?php wdnInclude("/wdn/templates_4.1/includes/search.html"); ?>
-				</div>
-			</div>
-			<div id="wdn_logo_lockup">
-				<div class="wdn-inner-wrapper">
-					<?php wdnInclude("/wdn/templates_4.1/includes/logo.html"); ?>
-						<span id="wdn_site_affiliation"><!-- TemplateBeginEditable name="affiliation" --><!-- TemplateEndEditable --></span>
-						<span id="wdn_site_title"><!-- TemplateBeginEditable name="titlegraphic" -->VITA Lincoln<!-- TemplateEndEditable --></span>
-				</div>
-			</div>
-		</header>
-		<div id="wdn_navigation_bar" class="wdn-band">
-			<nav id="breadcrumbs" class="wdn-inner-wrapper" role="navigation" aria-label="breadcrumbs">
-				<!-- TemplateBeginEditable name="breadcrumbs" -->
-				<ul>
-					<li><a href="https://www.unl.edu/" title="University of Nebraska&ndash;Lincoln" class="wdn-icon-home">UNL</a></li>
-					<li><a href="/" title="VITA Lincoln">VITA Lincoln</a></li>
-					<li>Not Found</li>
-				</ul>
+<body class="@@(_document['class'])@@ unl" data-version="5.0">
+<?php wdnInclude("/wdn/templates_5.0/includes/global/skip-nav.html"); ?>
+<header class="dcf-header" id="dcf-header" role="banner">
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/header-global-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-global-1.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/visit-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="visitlocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/visit-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/visit-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="applylocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/apply-local.html"); ?>
 				<!-- TemplateEndEditable -->
-			</nav>
-			<div id="wdn_navigation_wrapper">
-				<nav id="navigation" role="navigation" aria-label="main navigation">
-					<!-- TemplateBeginEditable name="navlinks" -->
-					<?php include "$root/sharedcode/navigation.php"; ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="givelocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/give-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-2.html"); ?>
```

Comment:
What is this `visit`, `apply`, `give` pattern happening here?

Reply:
I'm not sure I understand the purpose of these files tbh. I just kept them in there because they were in the original template files. Perhaps they don't have a functional purpose today, but there is an intended use for them in the future? I'm not sure.

FYI, they all appear to be blank files. e.g. view-source:http://unlcms.unl.edu/wdn/templates_5.0/includes/local/give-local.html

You can check out the other files in the folder here if you want...
http://unlcms.unl.edu/wdn/templates_5.0/includes/

Reply:
However, the pages don't appear to actually be making these requests, so I don't think it is much of an issue.